### ISSUE TITLE
Data Preprocessing

### ISSUE THREAD

**Issue by tomanick** [SymenYang/CPCGAN#3]
Thanks for your sharing!
I have downloaded shapenet dataset, but their format is ".txt".
![shapenet](https://user-images.githubusercontent.com/37836335/171321328-06b96f82-bdec-4aac-b59f-63d11c9ddb02.png)
![shapenet2](https://user-images.githubusercontent.com/37836335/171321336-f6207bc2-3b32-4c9f-ab46-938106913229.png)
When I wanted to do "bash DataPreprocessing.sh", there was an error said that the format should be .pts.
So, should I change the original .txt format to .pts format for train_data and .seg format for train_label ?
How should I do this change? 
I do not know how to change .txt format to .pts format and .txt format to .seg format in python.

**Comment by tomanick** [comment]
I have solved this problem. 
Afterwards, I found that I could use os.rename to change ".txt" format to ".pts" format.

In addition, when I started to train a model, I found that the result was weird.
![training](https://user-images.githubusercontent.com/37836335/171579452-3f9237b1-229a-431e-853d-0111fe1bbdc0.png)

The result was 10000?
It seemed that the metric was not accuracy.
Because I want to fine-tune more hyperparameters, is this possible to train by using python file like train.py rather than DLNest? @SymenYang 
Thanks!

**Comment by SymenYang** [comment]
I downloaded the data from https://shapenet.cs.stanford.edu/iccv17/. It seems no batch of data has ever been fed into the model, as no lines are output between different epochs. So please check whether the input data items exist. I would suggest you fine-tune hyperparameters within the framework of DLNest by changing the configs or changing the codes. And you can also write an easy script to do the model initialization (including model init and optimizer init) and feed data into the runOneStep function by yourself.

**Comment by yogkul2000** [comment]
> I have solved this problem. Afterwards, I found that I could use os.rename to change ".txt" format to ".pts" format.
> 
> In addition, when I started to train a model, I found that the result was weird. ![training](https://user-images.githubusercontent.com/37836335/171579452-3f9237b1-229a-431e-853d-0111fe1bbdc0.png)
> 
> The result was 10000? It seemed that the metric was not accuracy. Because I want to fine-tune more hyperparameters, is this possible to train by using python file like train.py rather than DLNest? @SymenYang Thanks!

Hi,
Were you able to fix this?
